src/Symfony/Component/RateLimiter/ClockTrait.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RateLimiter;
+
+use Psr\Clock\ClockInterface;
+
+/**
+ * @internal
+ */
+trait ClockTrait
+{
+    private ?ClockInterface $clock;
+
+    public function setClock(?ClockInterface $clock): void
+    {
+        $this->clock = $clock;
+    }
+
+    public function now(): float
```

Comment:
should this we public ? It introduces a public method in each class using this trait, while it think it could be private.

Reply:
indeed